src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -315,6 +320,9 @@ def app_deploy(cmd, client, resource_group, service, name,
 
     file_type, file_path = _get_upload_local_file(jar_path)
 
+    if runtime_version == "NetCore_31":
```

Comment:
Just a quick question, can we reuse any enumerations defined in the SDK (e.g. `_app_platform_management_client_enums.py`) instead of raw string here?